src/entitysdk/models/entity.py

```diff
@@ -14,7 +14,7 @@ class Entity(Identifiable):
 
     createdBy: Annotated[
         Agent | None,
-        Field(description="The agent that created this entity."),
+        Field(description="The agent that created this entity.", discriminator="type"),
```

Comment:
The discriminator is already set in the Agent annotation, does it need to be repeated here?

Reply:
Indeed. Removed it.

Reply:
Not removed yet :)

Reply:
Sorry about that. It slipped back. Removed it now.